### ISSUE TITLE
Gopkg.lock have a issue about openapi3filter

### ISSUE THREAD

**Issue by e346m** [danielgtaylor/apisprout#18]
Thank you for developing apisprout!!!  It's really helpful :)

I found Gopkg.toml setting doesn't catch up with kin-openapi update
update contents is like https://github.com/getkin/kin-openapi/commit/fa639d0d1f72da2df88b96f689909fbcb8a2412d

when I load openapi.json, apisprout cause panic like blow
```
panic: Validating Swagger failed: Security scheme of type 'apiKey' should have 'in'. It can be 'query' or 'header', not 'cookie'
```
kin-openapi  have already handle https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#securitySchemeObject, but latest apisprout image still give above messeage...

So I clone this repository and remove Gopkg.lock and build docker on the local machen then it works
I think I might have not hanlde this issue by removing lock file.  I'm not tottaly familier with go and go dependency though.

Sorry for just repoting but I post it because I feel it helps someone If they have encounted same problem. 

By the way I really appliciate apisprout!!

**Comment by danielgtaylor** [comment]
I will upgrade the `kin-openapi` library as part of #20, hopefully that fixes this as well.

**Comment by danielgtaylor** [comment]
This should be fixed in 43648d4
